<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormImageView</class>
 <widget class="QWidget" name="FormImageView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollAreaZoom" name="scrollAreaMain">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="FormImageRegion" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>136</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderScale">
       <property name="minimum">
        <number>-49</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonScaleHome">
       <property name="toolTip">
        <string>Вернуться к исходному масштабу</string>
       </property>
       <property name="text">
        <string>100%</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonScaleBest">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonScaleOut">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonScaleIn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionScaleIn">
   <property name="icon">
    <iconset resource="Ui.qrc">
     <normaloff>:/Icons/ZoomIn.png</normaloff>:/Icons/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
   <property name="toolTip">
    <string>Увеличить изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionScaleOut">
   <property name="icon">
    <iconset resource="Ui.qrc">
     <normaloff>:/Icons/ZoomOut.png</normaloff>:/Icons/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
   <property name="toolTip">
    <string>Уменьшить изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionScaleHome">
   <property name="text">
    <string>100%</string>
   </property>
   <property name="toolTip">
    <string>Вернуться к исходному масштабу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionScaleBest">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ui.qrc">
     <normaloff>:/Icons/Best fit.png</normaloff>:/Icons/Best fit.png</iconset>
   </property>
   <property name="text">
    <string>Вместить</string>
   </property>
   <property name="toolTip">
    <string>Уместить всё изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormImageRegion</class>
   <extends>QWidget</extends>
   <header>FormImageRegion.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QScrollAreaZoom</class>
   <extends>QScrollArea</extends>
   <header>QScrollAreaZoom.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Ui.qrc"/>
 </resources>
 <connections/>
</ui>
